ksql-execution/src/main/java/io/confluent/ksql/execution/timestamp/TimestampExtractionPolicyFactory.java

```diff
@@ -13,7 +13,7 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package io.confluent.ksql.util.timestamp;
+package io.confluent.ksql.execution.timestamp;
```

Comment:
doesn't it make more sense for this to belong in the KS specific ksql-streams module, given its dealing with KS types?  

Same for the actual extraction policies themselves?

In fact, do we even need the extraction policies now? Can we not just convert the `TimestampColumn` straight into a `TimestampExtractor`?

Reply:
Good call

Reply:
>In fact, do we even need the extraction policies now? Can we not just convert the TimestampColumn straight into a TimestampExtractor?

Yeah, I noticed this too. Left it out of this patch though.